src/test/regress/expected/multi_create_table.out

```diff
@@ -360,6 +354,48 @@ SELECT repmodel FROM pg_dist_partition WHERE logicalrelid='repmodel_test'::regcl
 
 DROP TABLE repmodel_test;
 RESET citus.replication_model;
+-- Test initial data loading
+CREATE TABLE data_load_test (col1 int, col2 text);
+INSERT INTO data_load_test VALUES (132, 'hello');
+INSERT INTO data_load_test VALUES (243, 'world');
+-- create_distributed_table copies data into the distributed table
+SELECT create_distributed_table('data_load_test', 'col1');
```

Comment:
Also, I forget; is there anything transaction-related to be testing here, or do all "create distributed table" methods immediately fail within a transaction?